<?xml version="1.0"?>
<launch>
	<arg name="TestDevice" value="Joy"/>
	<node pkg="masterslave" type="controldevice" name="$(arg TestDevice)" args="$(arg TestDevice)" output="screen">
		<param name="rotGain" value="20.0"/>
		<param name="transGain" value="20.0"/>
		<param name="button/4" type="string" value="close_gripper"/>
		<param name="button/5" type="string" value="open_gripper"/>
	</node>
	<node respawn="true" pkg="joy" type="joy_node" name="joystick">
		<param name="dev" type="string" value="/dev/input/js1" />
		<param name="deadzone" value="0.12"/>
		<param name="autorepeat_rate" value="60"/>
	</node>
	<node pkg="masterslave" type="masterslave" name="MasterSlave" args="$(arg TestDevice)" output="screen">
		<param name="gripper_vel" value="20"/>
	</node>

</launch>

